src/rbd_fuse/rbd-fuse.cc

```diff
@@ -892,6 +892,7 @@ int
 connect_to_cluster(rados_t *pcluster)
 {
 	int r;
+	g_ceph_context->_conf.finalize_reexpand_meta();
```

Comment:
I have not checked if it makes any difference but probably it is better to have this after `global_init_postfork_start`, as we do for other cases?

Reply:
Yeah, it is. As the last fixing in rbd-nbd, without the reexpanding, we cannot to figure out which log file should it be. So in any case when forking we should reexpand the meta in the child.